src/web.ts

```diff
@@ -57,10 +57,14 @@ export class HttpPluginWeb extends WebPlugin implements HttpPlugin {
 
     if (contentType.indexOf('application/json') === 0) {
       req['body'] = JSON.stringify(options.data);
-    } else if (contentType.indexOf('application/x-www-form-urlencoded') === 0) {
-    } else if (contentType.indexOf('multipart/form-data') === 0) {
-    }
-
+    } else if ((contentType.indexOf('application/x-www-form-urlencoded') === 0) ||
```

Comment:
You do not need the else here if you return in the first if early. Would make it more readable.

I realise it is not part of this MR, so would understand keeping it as it was.

Reply:
You can use _String.prototype.includes()_ then you don't have to handle indexes.

Reply:
haha... why did i listen to you ;-). I will put it back